File: public/phase-2-20160620022203/JAVASCRIPT/greedy-man.js
// Name : Greedy Man
// input:
// n :  the number of devices that will fit into
// m :  the number of smartphones available for sale
// a : 
var index,result;
var elements = readline().split(" ");
var n = parseInt(elements[0], 10);
var m = parseInt(elements[1], 10);
var a = [];
var a_elements = readline().split(' ');
for(index=0;index<n;index++)
	a.push(parseInt(a_elements[index], 10));


// write your code here
// store your results in `result`

// Dummy Data
result = 8;


// output
print(result);

File: public/phase-2-20160620022203/C_PLUS_PLUS/greedy-man.cpp
// Name : Greedy Man
#include <iostream>
#include <string>
using namespace std;

int main(int argc, char const *argv[]) {
	// input:
	// n : the number of devices that will fit into
	// m : e number of smartphones available for sale
	// a : price of a smartphones
	int index;
	int n, m;

	cin >> n >> m;
	int  a[m];

	for(index=0;index<m;index++)
		cin >> a[index];


	// write your code here
	// store your results in `maximum_sum_of_money`

	// output
	// Dummy Data
	int maximum_sum_of_money=8;

	cout << maximum_sum_of_money;

}